### PR TITLE
Changed noromax URL

### LINKED ISSUE #16367: Change url Noromax
### Source information

Noromax 1.4.6

### Source language

Indonesian 

### Source new URL

https://noromax.my.id/

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source URL is not updated yet.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by heartlog**
Checklist:
Closes #16367 
- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/zeistmanga/ZeistMangaGenerator.kt [hunk 1]
```diff
@@ -9,7 +9,7 @@ class ZeistMangaGenerator : ThemeSourceGenerator {
 
     override val themeClass = "ZeistManga"
 
-    override val baseVersionCode: Int = 6
+    override val baseVersionCode: Int = 7
```
Comment (alessandrojean) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/zeistmanga/ZeistMangaGenerator.kt:12]: This isn't the variable you need to bump. By bumping this you will cause all extensions in the generator to be updated unnecessarily.

```suggestion
    override val baseVersionCode: Int = 6
```

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/zeistmanga/ZeistMangaGenerator.kt [hunk 2]
```diff
@@ -18,7 +18,7 @@ class ZeistMangaGenerator : ThemeSourceGenerator {
         SingleLang("KLManhua", "https://klmanhua.blogspot.com", "id", isNsfw = true),
         SingleLang("Manga Ai Land", "https://manga-ai-land.blogspot.com", "ar"),
         SingleLang("Muslos No Sekai", "https://muslosnosekai.blogspot.com", "es"),
-        SingleLang("Noromax", "https://www.noromax.xyz", "id"),
+        SingleLang("Noromax", "https://noromax.my.id", "id"),
```
Comment (alessandrojean) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/zeistmanga/ZeistMangaGenerator.kt:21]:
```suggestion
        SingleLang("Noromax", "https://noromax.my.id", "id", overrideVersionCode = 1),
```

**Review by alessandrojean — CHANGES_REQUESTED** [review]

**Comment by heartlog** [comment]
@alessandrojean is requested to review the changes.

**Comment by heartlog** [comment]
> Be honest have you tested any of these changes? I doubt it highly

I wonder

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/zeistmanga/ZeistMangaGenerator.kt (hunk 1 above)
Comment (alessandrojean) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/zeistmanga/ZeistMangaGenerator.kt]: It applies to the multisrc ones, please do read the CONTRIBUTING.md guide carefully.

**Comment by alessandrojean** [comment]
What do you mean by "I wonder"? It's a simple yes/no question. Have you really tried this modifications on Android Studio or you just edited the files through GitHub website? The branches named `patch` make it obvious. Be honest, otherwise the PRs might get closed without merging.

**Comment by CarlosEsco** [comment]
Do your stuff in the IDE, then open a PR when you actually verified everything

**Pull request closed without merging**